### DISCUSSION TITLE
GitHub Copilot could not connect to server

### DISCUSSION THREAD

**Discussion by caotianchao** [community/community#17413] [Copilot Conversations] [answered]
[ERROR] [default] [2022-05-28T01:40:31.408Z] GitHub Copilot could not connect to server. Extension activation failed: "connect ECONNREFUSED 127.0.0.1:443"

but just now, it shows me that I can connect copilot by my Github account

**Comment by kitt1987** [comment]
Try a public DNS server instead of that from your telecom provider.

**Reply by caotianchao** [reply]
Thanks, I will try again.

**Reply by caotianchao** [reply]
GitHub Copilot could not connect to the server. **Extension activation failed: "Cancelled"**

**Comment by davecheney** [comment, marked as answer]
It looks like you're using some kind of VPN software which has rewritten all your DNS lookups to be localhost.

**Reply by caotianchao** [reply]
that's right, I had to use VPN to connect to GitHub copilot, but I can't log in to Visual studio code to use GitHub copilot.

**Reply by caotianchao** [reply]
Any suggestions?

**Comment by caotianchao** [comment]
thanks

Kitt Hsu ***@***.***> 于2022年5月29日周日 11:45写道：

**Comment by Sagners** [comment]
So,what is the final answer?

**Comment by Nunchakus888** [comment]
I resolved it, see follow the link.
https://github.com/orgs/community/discussions/51076#discussioncomment-9075622
